checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -27,3 +27,25 @@ class Config {
         return [Config]::new($use_rcc)
     }
 }
+function CreateCommand([Config]$config) {
+    $parentDir = Split-Path $PSScriptRoot -Parent
+
+    if ($config.rcc) {
+        $rccPath = Join-Path $PSScriptRoot "rcc.exe"
+        $robotPath = Join-Path $parentDir "config\robot.yaml"
+
+        $argumentsList = "run --robot $robotPath"
+    }
+    else {
+        $pythonExe = (Get-Command python).Source
+        $moduleName = "robotmk.scheduler"
+        $configPath = Join-Path $parentDir "config\robotmk.json"
+
+        $argumentsList = "-m $moduleName --config $configPath"
+    }
+
+    return [PSCustomObject]@{
+        Executable = if ($config.rcc) { $rccPath } else { $pythonExe }
```

Comment:
We can just define a variable `$executable` above (which is either `Join-Path $PSScriptRoot "rcc.exe"` or `(Get-Command python).Source`), then we don't have to check any condition here.